setup.sh

```diff
@@ -9,6 +9,7 @@ sudo useradd mylar -u 13005
 sudo useradd prowlarr -u 13006
 sudo useradd qbittorrent -u 13007
 sudo useradd jackett -u 13008
+sudo useradd overseerr -u 13010
```

Comment:
I think 13009 is more appropriate here right?

Reply:
Sure I can change that